recipes/roocs-grids/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "roocs-grids" %}
+{% set package_name = "roocs_grids" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ package_name }}-{{ version }}.tar.gz
+  sha256: 7dbf178314950b0c0160032412677141106701c38141d2481c0e622c7b3efed0
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python
```

This should be the same as in run.
```suggestion
    - python >=3.8
```